### PR TITLE
Whcm fixes

### PR DESCRIPTION
**Pull request by jnurthen**
partial resolution for #2529

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [x] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [x] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Test the following stories in WHCM 
* accordion
* actionbar
* avatar
* breadcrumbs
* tooltip
* toast
* tags
* statuslight
* sidenav
* progressCircle
* progessBar
* dialog
* tabs

| Component | Screenshot B4 | Screenshot After |
| --------------- | --------------- | --------------- |
| Accordion | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_50_26 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_50_17 PM |
| ActionBar | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_52_25 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_52_13 PM |
| Avatar | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_53_11 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_53_03 PM |
| Breadcrumbs | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_54_06 PM  | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_53_54 PM  |
| Tooltip | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_54_25 PM  | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_54_44 PM |
| Toast | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_55_11 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_55_20 PM |
| Tags | TagGroup - onRemove ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 5_11_36 PM |  TagGroup - onRemove ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 5_11_18 PM|
| Statuslight | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_56_04 PM (1)  | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_55_50 PM |
| Sidenav | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_56_50 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_56_42 PM  |
| ProgressCircle | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_57_30 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_57_17 PM |
| ProgressBar | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_58_11 PM  | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_58_01 PM |
| Dialog | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_58_46 PM | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_58_37 PM |
| Tabs | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_59_39 PM  | Accordion - disabledKeys_ files, shared ⋅ Storybook and 1 more page - Work - Microsoft​ Edge 11_30_2021 4_59_31 PM |

### CODE REVIEW

File: packages/@adobe/spectrum-css-temp/components/accordion/skin.css [hunk 1]
```diff
@@ -62,3 +62,10 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Accordion-itemHeader {
+    &:focus-ring {
+      outline: 3px solid CanvasText;
```
Comment (majornista) [inline packages/@adobe/spectrum-css-temp/components/accordion/skin.css:68]: How is the focus style drawn for `.spectrum-Accordion-itemHeader` without WHCM? Should this be?:
```css
@media (forced-colors: active) {
  .spectrum-Accordion-itemHeader {
    &:focus-ring {
      &:after  {
        background-color: CanvasText;
      }
    }
  }
}
```

**Comment by majornista** [comment]
Are tooltip example images reversed?

**Comment by jnurthen** [comment]
> Are tooltip example images reversed?

They were. Fixed the screenshots.

File: packages/@adobe/spectrum-css-temp/components/accordion/skin.css (hunk 1 above)
Comment (jnurthen) [reply packages/@adobe/spectrum-css-temp/components/accordion/skin.css:68]: without WHCM it is a border at the left hand side. Really too subtle for WHCM. IMO it is not enough which is why I added the outline.

**Review by majornista — APPROVED** [review]

File: packages/@adobe/spectrum-css-temp/components/tags/skin.css [hunk 2]
```diff
@@ -155,3 +155,50 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-Tags-item {
```
Comment (LFDanLu) [inline packages/@adobe/spectrum-css-temp/components/tags/skin.css:160]: Looks like the Tags are missing a distinguishable active style in WHCM. Clicking on the Tag in the "icon, removable" story in non-WHCM displays a style change but not in WHCM

**Review by LFDanLu — CHANGES_REQUESTED** [review]
For the most part looks good to me, just one change really required for the Tag active styling. @snowystinger perhaps we should do the same to the breadcrumbs and tabs css that we did for the other PR? IMO we can do that in a follow up since the css files for breadcrumbs and tabs isn't as complex

File: packages/@adobe/spectrum-css-temp/components/accordion/skin.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (snowystinger) [reply packages/@adobe/spectrum-css-temp/components/accordion/skin.css:68]: that is typically our preference, i do not remember why at this time though
but if you mix outline and box shadow you can get some odd results

**Comment by snowystinger** [comment]
@LFDanLu followup for it is fine with me now that we have an example to go off of

**Review by devongovett — APPROVED** [review]

**Pull request merged**